nox.py

```diff
@@ -30,13 +30,13 @@ def docs(session):
     # Install Sphinx and also all of the google-cloud-* packages.
     session.chdir(os.path.realpath(os.path.dirname(__file__)))
     session.install('Sphinx >= 1.6.2', 'sphinx_rtd_theme')
+    session.install('.')
```

Comment:
`session.install('--no-deps', '.')` works, but turns into `pip install --upgrade --nodeps .`, which seems to install all the dependencies anyway. :(